### ISSUE TITLE
Java to CPP translation : vector -- set (int index, E element)

### ISSUE THREAD

**Issue by lublud** [darrivau/GOOL#11]
```java
list.set(0, "tata");
```
is translated into:
```cpp
list -> set(0, ( new std::string ( "tata" ) ));
```

However, set (int index, E element) does not exist in C++ for vector.
Suggestion for translation:
```cpp
list->at(0) = new std::string("tata");
```

**Comment by darrivau** [comment]
GoolTestTypeListToVectorCpp - TestSetRemove :  std::vector has no member named ‘set’

**Comment by darrivau** [comment]
Corrected in current commit (887db7d) with `list->erase(std::find(list->begin(),list->end(),"tata"));`
